services/QuillLMS/spec/queries/snapshots/top_concepts_practiced_query_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module Snapshots
+  describe TopConceptsPracticedQuery do
+    include_context 'Snapshots Period CTE'
+
+    context 'big_query_snapshot', :big_query_snapshot do
+      let(:num_classrooms) { 1 }
+      let(:max_activity_session_count) { 11 }
+      let(:num_concepts) { 11 }
+
+      let(:concepts) { create_list(:concept, num_concepts) }
+      let(:activities) { concepts.map { |concept| create(:activity, data: {modelConceptUID: concept.uid}) } }
+      let(:classroom_units) { classrooms.map { |classroom| create(:classroom_unit, classroom: classroom) } }
+
+      # We have one activity connected to each concept.
+      # We want to create an activity_session for each activity, then a second activity_session for all but one activity, and so on.  The outcome we desire is a unique number of activity_sessions tied to each activity
+      let(:activity_session_bundles) do
+        activities.map.with_index do |activity, activity_index|
+          create_list(:activity_session, num_concepts - activity_index, activity: activity, classroom_unit: classroom_units.first)
+        end
+      end
+
+      let(:runner_context) {
+        [
+          classrooms,
+          teachers,
+          classrooms_teachers,
+          schools,
+          schools_users,
+          concepts,
+          activities,
+          classroom_units
+        ]
+      }
+
+      let(:runner) { QuillBigQuery::TestRunner.new(cte_records) }
```

Comment:
`let(:runner)` and `let(:results)` can be removed since they are already defined in the shared_context [here](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/spec/support/shared_contexts/quill_big_query_test_runner_setup.rb#L7)